src/lib.rs

```diff
@@ -65,6 +66,7 @@ mod public_key;
 pub mod request;
 pub mod response;
 
+pub use coins::Coins;
```

Comment:
Can we separate the `pub use` from the `use` statements? It's hard for me to tell what is being publicly exported.

Reply:
Raised https://github.com/maidsafe/safe-nd/pull/23